web/src/ui/roomview/RoomPreview.css

```
@@ -18,6 +18,8 @@ div.room-view.preview > div.preview-inner {
 
 	> h2.room-name {
 		margin: 0;
+		max-height: 6rem;
+		overflow-y: scroll;
```

windows and chrome users have visible scrollbars even if there's nothing to scroll so can't use that (firefox on linux and safari have overlay scrollbars)

```suggestion
		overflow-y: auto;
```